### PR TITLE
Added info about stage branches

### PR DESCRIPTION
**Pull request by vikram-redhat**

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@openshift/team-documentation PTAL. 

No rev history needed.

File: contributing_to_docs/create_or_edit_content.adoc [hunk 1]
```diff
@@ -152,7 +152,7 @@ work.
...
+3. Attach labels to your *Pull Request* to indicate to the DOCS team which branches your PR might apply to. Some requests may only apply to the origin distro (origin-only), and some may only apply to a specific version of the enterprise (branch/enterprise-3.0, branch/enterprise-3.1, branch/enterprise-3.2, branch/enterprise-3.3) or dedicated (branch/dedicated) or online (branch/online) docs. If a request applies to multiple versions of a distro, then specify the minimum version (branch/enterprise-3.1 and above, for example). The docs team maintains stage branches for all active and future distros and your PR will be applied to one or more these stage branches before getting published to the production branch.
```
Comment (mburke5678) [inline contributing_to_docs/create_or_edit_content.adoc]: >>(branch/enterprise-3.0, branch/enterprise-3.1, branch/enterprise-3.2, branch/enterprise-3.3) 
Do we want to update these versions to the active versions?

**Comment by vikram-redhat** [comment]
@mburke5678 - yup makes sense. I have updated the PR

**Comment by vikram-redhat** [comment]
Gonna merge as the errors in Travis were to do with Travis being down.

**Pull request merged**